bciers/libs/actions/src/api/getBusinessStructures.ts

```diff
@@ -0,0 +1,11 @@
+import { actionHandler } from "@bciers/actions";
```

Comment:
I struggled with what to name this. Is`actions/get` better for these? Either way I would like to start putting a lot of these in here to reuse.